omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PersonController1_9Test.java

```diff
@@ -82,6 +82,23 @@ public void shouldGetAPersonByUuid() throws Exception {
 		assertNull(PropertyUtils.getProperty(result, "auditInfo"));
 	}
 	
+	@Test
+	public void shouldGetALinkIfPersonIsPatient() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		List<Object> links = (List<Object>) result.get("links");
+		String expectedLink = RestConstants.URI_PREFIX + "v1/patient/" + RestTestConstants1_8.PATIENT_UUID;
+		String recievedLink = null;
+		for (Object link : links.toArray()) {
+			if ("patient".equals((String) PropertyUtils.getProperty(link, "rel"))) {
+				recievedLink = (String) PropertyUtils.getProperty(link, "uri");
+			}
+		}
+		assertNotNull(recievedLink);
+		assertEquals(expectedLink, recievedLink);
```

Comment:
Can you also test for cases where Person is not a Patient?

Reply:
Testing for possibly of this has 0% code coverage and hence I think is not necessary. The `If` clause I used was without doubt, explicit in only branching and executing block of of code in it that needs to be covered while unit testing otherwise nothing is executed, so no need to test nothing-executing.